tests/platform_tests/counterpoll/counterpoll_helper.py

```diff
@@ -0,0 +1,52 @@
+from tests.platform_tests.counterpoll.counterpoll_constants import CounterpollConstants
+
+
+class ConterpollHelper:
+    @staticmethod
+    def get_counterpoll_show_output(duthost):
+        return duthost.show_and_parse(CounterpollConstants.COUNTERPOLL_SHOW)
+
+    @staticmethod
+    def get_available_counterpoll_types(duthost):
+        available_option_list = []
+        COMMANDS = 'Commands:'
+        counterpoll_show = duthost.command(CounterpollConstants.COUNTERPOLL_QUEST)[CounterpollConstants.STDOUT]
+        index = counterpoll_show.find(COMMANDS) + len(COMMANDS) + 1
+        for line in counterpoll_show[index:].splitlines():
+            available_option_list.append(line.split()[0])
+        return [option for option in available_option_list if option not in CounterpollConstants.EXCLUDE_COUNTER_SUB_COMMAND]
+
+    @staticmethod
+    def get_parsed_counterpoll_show(counterpoll_show):
+        parsed_counterpoll = {}
+        for counterpoll in counterpoll_show:
+                parsed_counterpoll[counterpoll[CounterpollConstants.TYPE]] = {
```

Comment:
This indention is 8 whitespaces. It should be 4. This could raise exception.

Reply:
Fixed. Thank you @wangxin